components/cluster-secret-store-rh/base/app-sre-secret-store.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
```

Comment:
@gbenhaim Since this is a secretStore, should we create this in infra-deployments? Tenants can create secretstore in their NS either manually or using gitops in the tenants repo

Reply:
Having this in konflux-release-data would also allow me to specify which konflux instance I want this applied in. Would be fine for me too :)

Reply:
ok, let's put this in konflux-release-data, it make more sense since infra-deployments is the GitOps repo for Konflux itself, not the user payloads

Reply:
Got you. Will do :+1: